src/utils/signaling.js

```diff
@@ -1499,6 +1500,26 @@ Signaling.Standalone.prototype.processRoomParticipantsEvent = function(data) {
 				console.error('Unknown room participant event', data)
 			}
 		} else {
+			// Convert participant data in federated conversations
+			if (this.settings.federation?.nextcloudServer) {
+				for (let i = 0; i < data.event.update.users.length; i++) {
+					const user = data.event.update.users[i]
+					if (user.actorType === ATTENDEE.ACTOR_TYPE.USERS) {
```

Comment:
Also here. Must check for existance of the attributes so the desktop client doesn't break against 19

Reply:
Commit dropped.